apps/pipelines/nodes/helpers.py

```diff
@@ -0,0 +1,22 @@
+from contextlib import contextmanager
+
+from django.db import transaction
+
+from apps.chat.models import Chat
+from apps.experiments.models import ConsentForm, Experiment, ExperimentSession
+from apps.users.models import CustomUser
+
+
+@contextmanager
+def temporary_session(team):
+    """A temporary sesssion setup that is rolled back after the context exits."""
+    with transaction.atomic():
+        owner = CustomUser.objects.create_user(username="test", password="test")
+        consent_form = ConsentForm.objects.get(team=team, is_default=True)
```

Comment:
These are the minimum set of models that need to be present for these records to exist.